src/recovery/cubic.rs

```diff
@@ -61,9 +61,13 @@ const BETA_CUBIC: f64 = 0.7;
 
 const C: f64 = 0.4;
 
-/// The packet count threshold to restore to the prior state if the
-/// lost packet count since the last checkpoint is less than the threshold.
-const RESTORE_COUNT_THRESHOLD: usize = 10;
+/// The packet count threshold (%) in relative to cwnd, to roll back to
+/// the prior state if the lost packet count since the last checkpoint
+/// is less than the threshold.
```

Comment:
I find this sentence hard to parse but have no suggestion on improvements.

Reply:
I added a suggestion. I don't think we need to repeat the whole thing, since the mechanism is described in the code where the constant is used.